### DISCUSSION TITLE
In MIMIC-IV, what is the difference between 'Heparin Sodium (Prophylaxis)' and 'Heparin Sodium' in mimiciv_icu.d_items?

### DISCUSSION THREAD

**Discussion by dbcoffee** [MIT-LCP/mimic-code#1735] [MIMIC-IV] [answered]
In the d_items table, 'Heparin Sodium (Prophylaxis)' has the unit 'dose', while 'Heparin Sodium' has the unit 'units'. How many units are equal to 1 dose? Under what circumstances would a patient be administered 'Heparin Sodium (Prophylaxis)' versus 'Heparin Sodium'?

**Comment by alistairewj** [comment]
Hm maybe try cross-referencing with the prescriptions / eMAR table to figure out how many units are in a dose. It's probably a standard formulation - I don't want to hazard a guess at what though.

**Reply by dbcoffee** [reply]
Which field should I use to connect it with the prescriptions/eMAR table? Is it the 'orderid' in mimiciv_icu.inputevents? And which field should it correspond to in the prescriptions/eMAR table?

Please forgive me, I am a novice.

**Reply by alistairewj** [reply, marked as answer]
*prescriptions* and *emar* can be linked, and the documentation tries to highlight how (the "links to" section): https://mimic.mit.edu/docs/iv/modules/hosp/emar/

> Links to
> * emar_detail on emar_id
> * pharmacy on pharmacy_id
> * prescriptions on pharmacy_id
> * poe on poe_id

*inputevents* does not link to the hospital data. However you can do an approximate match using time of day, but they are documented separately, so they won't always perfectly match.
